libraries/AP_HAL/board/linux.h

```diff
@@ -8,13 +8,18 @@
 #define HAL_STORAGE_SIZE_AVAILABLE  HAL_STORAGE_SIZE
 
 // make sensor selection clearer
-#define PROBE_IMU_I2C(driver, bus, addr, args ...) ADD_BACKEND(AP_InertialSensor_ ## driver::probe(*this,hal.i2c_mgr->get_device(bus, addr),##args))
+#define PROBE_IMU_I2C(driver, bus, addr, args ...) ADD_BACKEND(AP_InertialSensor_ ## driver::probe(*this,GET_I2C_DEVICE(bus, addr),##args))
```

Comment:
Pointing out this one, but it's also for others below: this is missing the `std::move` call.

Side comment: I **think** this is probably fine and I would even go further: delete the copy assignment operator from OwnPtr and we should be able to remove most std::move calls - the compiler should then do a copy/move elision (which the std::move call prevents).